.github/workflows/ci.yml

```diff
@@ -5,56 +5,67 @@ on:
     branches:
       - "main"
   workflow_dispatch:
-    
+
 jobs:
   Spec:
     uses: "puppetlabs/cat-github-actions/.github/workflows/module_ci.yml@main"
+    with:
+      runs_on: "ubuntu-20.04"
     secrets: "inherit"
 
   Acceptance:
     needs: "Spec"
-    runs-on: self-hosted
+    runs-on: ubuntu-20.04
     steps:
-    - uses: actions/checkout@v1
-    - name: Build
+    - name: Checkout Source
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Activate Ruby 2.7
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: "2.7"
+        bundler-cache: true
+
+    - name: Bundle Install
       run: |
         gem install bundler
```

Comment:
I don't think the `gem install bundler` is needed, bundler is installed in the `Activate Ruby 2.7` step above